eng/Versions.props

```diff
@@ -92,7 +92,7 @@
     <MicrosoftVisualBasicVersion>10.1.0</MicrosoftVisualBasicVersion>
     <MicrosoftVisualStudioCallHierarchyPackageDefinitionsVersion>15.8.27812-alpha</MicrosoftVisualStudioCallHierarchyPackageDefinitionsVersion>
     <MicrosoftVisualStudioCodeAnalysisSdkUIVersion>15.8.27812-alpha</MicrosoftVisualStudioCodeAnalysisSdkUIVersion>
-    <MicrosoftVisualStudioCodingConventionsVersion>1.1.20180528.2</MicrosoftVisualStudioCodingConventionsVersion>
+    <MicrosoftVisualStudioCodingConventionsVersion>1.1.20191022.5</MicrosoftVisualStudioCodingConventionsVersion>
```

Comment:
So if this package was built October 22nd of 2019 that by date at least is well after the 16.0 ship...worrisome?